packages/docs/package.json

```diff
@@ -9,17 +9,22 @@
     "dist"
   ],
   "scripts": {
-    "build": "yarn vendor && yarn micromodal && yarn selectize && yarn prism && hexo g",
+    "build": "yarn vendor && yarn micromodal && yarn choices && yarn prism && hexo g",
```

Comment:
Nitpick: This entire block could use some cleaning up. Not necessarily for this PR, but might be good to do a `prebuild` script. This would be automatically be run when running `yarn build`

https://docs.npmjs.com/misc/scripts

Reply:
Wanna add a chore ticket for this/next sprint? Alternately, we can scrap a lot of this if/when we swap doc frameworks.

Reply:
Yeah ill log this and handle it!

Reply:
#442 logged